docs/en/operations/backup.md

```diff
@@ -356,6 +357,267 @@ But keep in mind that:
+                <secret_access_key>Your AWS Secret Key</secret_access_key>
+                <!-- highlight-end -->
+                <s3_max_single_part_upload_size>33554432</s3_max_single_part_upload_size>
+            </backup_disk_s3_plain>
+            <s3_backup_compact>
+                <type>s3_plain</type>
+                <!-- highlight-start -->
+                <!-- NOTE: /backup_compact/ is a name of BACKUP -->
+                <endpoint>https://s3-backup-for-read-only-table.s3.amazonaws.com/tables/backup_compact/</endpoint>
+                <access_key_id>Your AWS Access Key</access_key_id>
+                <secret_access_key>Your AWS Secret Key</secret_access_key>
+                <!-- highlight-end -->
+                <s3_max_single_part_upload_size>33554432</s3_max_single_part_upload_size>
+            </s3_backup_compact>
+            <s3_backup_wide>
+                <type>s3_plain</type>
+                <!-- highlight-start -->
+                <!-- NOTE: /backup_wide/ is a name of BACKUP -->
+                <endpoint>https://s3-backup-for-read-only-table.s3.amazonaws.com/tables/backup_wide/</endpoint>
+                <access_key_id>Your AWS Access Key</access_key_id>
+                <secret_access_key>Your AWS Secret Key</secret_access_key>
+                <!-- highlight-end -->
+                <s3_max_single_part_upload_size>33554432</s3_max_single_part_upload_size>
+            </s3_backup_wide>
+        </disks>
+        <policies>
+            <s3_backup_compact>
+                <volumes>
+                    <main>
+                        <disk>s3_backup_compact</disk>
+                    </main>
+                </volumes>
+            </s3_backup_compact>
+            <s3_backup_wide>
+                <volumes>
+                    <main>
+                        <disk>s3_backup_wide</disk>
+                    </main>
+                </volumes>
+            </s3_backup_wide>
+        </policies>
+    </storage_configuration>
+    <backups>
+        <allowed_disk>backup_disk_s3_plain</allowed_disk>
+    </backups>
+</clickhouse>
+```
+
+### Create a database of type ordinary
+
+Enable the use of ordinary databases:
+```sql
+set allow_deprecated_database_ordinary=1;
+```
+
+Create the database:
+```sql
+create database ordinary_db engine=Ordinary;
+```
+
+### Create a table
+
+This will create a table in the `ordinary_db` database that has five partitions:
+- there are 100 rows, with two columns `part` and `key`
+- `key` is set to the value provided by the function `numbers(100)`
+- `part` is set to the value `number % 5`
+- the table is partitioned on the column `part`
+
+```sql
+CREATE TABLE ordinary_db.compact
+ENGINE = MergeTree
+PARTITION BY part
+ORDER BY key AS
+SELECT
+    number % 5 AS part,
+    number AS key
+FROM numbers(100)
+```
+
+### Backup the table to S3
+```sql
+BACKUP TABLE ordinary_db.compact
+TO Disk('backup_disk_s3_plain', 'backup_compact')
+SETTINGS deduplicate_files = 0
```

Comment:
@azat
I just thought - maybe it's better to rename this setting while it's not in the documentation yet? Because besides the deduplication this setting also means empty files won't be stored and also it prohibits incremental backups. It's difficult to understand why that is so. Maybe we rename this setting to something like just `plain`, what do you think? Like this

```
BACKUP TABLE ordinary_db.compact
TO Disk('backup_disk_s3_plain', 'backup_compact')
SETTINGS plain = 1
```

And also just for usability maybe we should not allow to store a plain backup to a `s3` disk with a recomendation to use `s3_plain`. What do you think?

Reply:
I am going to mark this PR as Draft while we wait for a decision on the setting above related to deduplication and also for https://github.com/ClickHouse/ClickHouse/pull/41976 to be merged.

Reply:
>I just thought - maybe it's better to rename this setting while it's not in the documentation yet? Because besides the deduplication this setting also means empty files won't be stored and also it prohibits incremental backups. It's difficult to understand why that is so. Maybe we rename this setting to something like just plain, what do you think? Like this

@vitlibar I agree that right now setting does not fully describe the purpose, however the format is not `plain` I would say. I was thinking about this, and what do you think about `attachable` (ClickHouse can ATTACH) or `compatible` (ClickHouse disks compatible, means that it can be ATTACH'ed) ?

> And also just for usability maybe we should not allow to store a plain backup to a s3 disk with a recomendation to use s3_plain. What do you think?

Yep, this is a good idea! Making backups to regular S3 does not makes a lot of sense, and even not safe, since to restore this backup you need metadata from the local disk (and user, that do not know this, may assume that if BACKUP is done, then, everything is fine).

Reply:
Actually I like `compatible` more, but it may sounds too generic, which may not be a good thing.

Reply:
We called `s3_plain` disks as "plain" because they don't need metadata to understand under which name each file is stored in s3 storage. For this kind of backup it looks we have almost the same thing - because for such backups we don't need metadata in `.backup` to understand under which name each file is stored inside the backup. The ideas look very similar, that's why I suggested "plain".

I don't like `compatible - it's too generic, the word `attachable` looks better. However I think we can implement a new disk type which reads files from any kind of backup and thus with that improvement (after we'll implement it) every backup can be made attachable.

Reply:
Yeah, I understand the reasoning for `plain`, but I thought that it also too generic.
Anyway I don't have any other ideas, and I'm I don't like that match neither of mine proposal, so I'm with `plain`.